CHANGELOG.md

```diff
@@ -11,6 +11,35 @@ BREAKING CHANGES:
 * Minimum Kubernetes versions supported is 1.16+. [[GH-883](https://github.com/hashicorp/consul-helm/pull/883)]
 * Connect: `-enable-health-checks-controller`, `-health-checks-reconcile-period`, `-cleanup-controller-reconcile-period` have been removed
   and are no longer supported as the controllers have been replaced by the endpoints controller. [[GH-892](https://github.com/hashicorp/consul-helm/pull/899)]
+* Connect: Support transparent proxy. [[GH-905](https://github.com/hashicorp/consul-helm/pull/905)]
+  This feature allows users to reach other services on the Consul Service Mesh by using KubeDNS instead of using `localhost`
+  and enforces all inbound and outbound traffic within a pod to go through the Envoy proxy.
+  Please see Transparent Proxy docs for more information.
```

Comment:
So we just talked about a page under service mesh and kubernetes that starts off with the same content, and eventually the kubernetes page would have k8s specific stuff and the service mesh page would be more general. So linking to https://www.consul.io/docs/connect/transparent-proxy would probably work here.

Reply:
Will there be a separate page under the `Kubernetes` section? I think that one would be ideal since it's kubernetes related.

Reply:
Alright, I've updated to that link preemptively but we could change it later.